### ISSUE TITLE
There was an error validating the utterance. Something went wrong. We've been notified.

### ISSUE THREAD

**Issue by brenmczuken** [wit-ai/wit#2411]
After creating an initial app that worked OK I have created several test apps to try out some other things. 
All of them show errors with most operations such as "There was an error validating the utterance. Something went wrong. We've been notified."
I cant delete these test apps either as that errors. One of these test apps ID is 1345083552661465
I did create an initial app that doesn't have these issues and can't see that I am doing anything different on the test apps

**Comment by wangzhenict** [comment]
Hi @beemczuken , is the problem still ongoing? we fixed some problems and not sure if they are related to your case. if it is still there, can you elaborate how you operate and get the error message. thanks!

**Comment by brenmczuken** [comment]
Hi, Yes it is still happening. So within the wit.ai control panel on the understanding screen, I fill out my utterance, select the appropriate intents and entities and then click Train and Validate, but I get the attached message
![2022-04-19 09_44_32-Wit ai](https://user-images.githubusercontent.com/56307658/163963249-27016693-df5d-417e-bd8f-68055e99a4a1.png)
This happens also to things like trying to delete an app
![2022-04-19 09_49_21-Wit ai](https://user-images.githubusercontent.com/56307658/163965237-09c2d432-f382-4b2e-aa6e-328ab3327a29.png)
I have been having this issue on a number of test apps but not on the initial one I created which works well. I have checked the settings for the test apps that don't work against the app that does work and I can't see that I have done anything differently.
